packages/components/src/primitives/index.js

```diff
@@ -1 +1,2 @@
 export * from './svg';
+export * from './horizontal-rule';
```

Comment:
We also need to include a note about this new component in [CHANGELOG](https://github.com/WordPress/gutenberg/blob/8616d45ea2a4a34b59540bed8b7f032e8e2f3466/packages/components/CHANGELOG.md).